Sources/MapboxDirections/DirectionsOptions.swift

```diff
@@ -369,6 +370,94 @@ open class DirectionsOptions: Codable {
         return queryItems
     }
     
+    /**
+     :nodoc:
+     Method to deserialize options parameters from given `url`.
+     */
+    open func restore(from url: URL) {
```

Comment:
This is generally useful functionality that application would find useful when implementing Handoff, so we should make it a formal part of the public API.